Dockerfile.deployment

```diff
@@ -65,7 +64,7 @@ ARG GIT_SHA
 RUN printf "%s" "$GIT_SHA" > version
 
 RUN useradd -m osuweb
-RUN chown -R osuweb bootstrap/cache storage /var/lib/nginx
+RUN chown -R osuweb bootstrap/cache storage public/uploads public/uploads-avatar public/uploads-replay
```

Comment:
I don't think the upload directories should be writable at all. Those will be gone if the docker instance is stopped. And mounted volumes don't use the existing permission.

Reply:
i probably did it for a good reason. will double-check

Reply:
was definitely a testing left-over, it's not needed. at least not under Kubernetes. can re-add later if needed. good catch